### PR TITLE
fix url for CN

### LINKED ISSUE #140: load_map=True results in URLError
This works just fine:

```
import sc2reader
path = "/home/volker/Sync/DatenVolker/git/sc2stats/zipout/CN/Group A/1 TIME vs BreakingGG/Time vs BreakingGG G1.SC2Replay"
sc2reader.load_replay(path)
```
but this

```
#!/usr/bin/env python
import sc2reader

path = "/home/volker/Sync/DatenVolker/git/sc2stats/zipout/CN/Group A/1 TIME vs BreakingGG/Time vs BreakingGG G1.SC2Replay"
sc2reader.load_replay(path, load_map=True)
```

fails with the message
```
Traceback (most recent call last):
  File "/usr/lib/python3.9/urllib/request.py", line 1342, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/usr/lib/python3.9/http/client.py", line 1255, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.9/http/client.py", line 1301, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.9/http/client.py", line 1250, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.9/http/client.py", line 1010, in _send_output
    self.send(msg)
  File "/usr/lib/python3.9/http/client.py", line 950, in send
    self.connect()
  File "/usr/lib/python3.9/http/client.py", line 921, in connect
    self.sock = self._create_connection(
  File "/usr/lib/python3.9/socket.py", line 822, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/usr/lib/python3.9/socket.py", line 953, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/volker/Sync/DatenVolker/git/sc2stats/./extract.py", line 5, in <module>
    sc2reader.load_replay(path, load_map=True)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 85, in load_replay
    return self.load(Replay, source, options, **new_options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 143, in load
    return self._load(cls, resource, filename=filename, options=options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 152, in _load
    obj = cls(resource, filename=filename, factory=self, **options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/resources.py", line 307, in __init__
    self.load_map()
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/resources.py", line 447, in load_map
    self.map = self.factory.load_map(self.map_file, **self.opt)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 105, in load_map
    return self.load(Map, source, options, **new_options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 142, in load
    resource, filename = self._load_resource(source, options=options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 201, in _load_resource
    contents = self.load_remote_resource_contents(resource, **options)
  File "/home/volker/.local/lib/python3.9/site-packages/sc2reader/factories/sc2factory.py", line 185, in load_remote_resource_contents
    return urlopen(resource).read()
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 517, in open
    response = self._open(req, data)
  File "/usr/lib/python3.9/urllib/request.py", line 534, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 1371, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/lib/python3.9/urllib/request.py", line 1345, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno -2] Name or service not known>
```

I'm using Python 3.9.1. Both ipv4 and ipv6 internet is working.

### PR DESCRIPTION
**Pull request by HADB**
fix #140 

There are some more url templates in `resources.py` like this. If needed, I'll try to fix them too.

### CODE REVIEW

**Comment by HADB** [comment]
[20210904_bean.SC2Replay.zip](https://github.com/ggtracker/sc2reader/files/7118914/20210904_bean.SC2Replay.zip)
Uploaded CN SC2Replay for testing.

File: sc2reader/resources.py [hunk 1]
```diff
@@ -1021,8 +1017,6 @@ class GameSummary(Resource):
     that the data is not necessarily in the places we expect.
     """
 
-    url_template = "https://{}-s2-depot.classic.blizzard.com/{}.s2gs"
```
Comment (StoicLoofah) [inline sc2reader/resources.py:1024]: Was this `url_template` unused, or am I not seeing the reference to it in the new code?

**Review by StoicLoofah — APPROVED** [review]
Looks good! The refactor is also nice. Just one question before I merge it in

File: sc2reader/resources.py (hunk 1 above)
Comment (HADB) [reply sc2reader/resources.py:1024]: This was unused.

**Pull request merged**